### ISSUE TITLE
Check Functionality for Mid-Project Release

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#77]
## User story
1. As a user
2. I want a software
3. So that has no severe bugs and fulfills its core functionality

## Acceptance criteria
* [ ] Test usability of frontend 
* [ ] Test completeness and correctness of backend API
* [ ] Test integration of frontend and backend

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by phildree** [comment]
Only high level looking (for errors)
